src/prefect/client/orchestration.py

```diff
@@ -3461,6 +3462,19 @@ async def raise_for_api_version_mismatch(self) -> None:
                 f"Major versions must match."
             )
 
+    async def update_flow_run_labels(
+        self, flow_run_id: UUID, labels: KeyValueLabelsField
+    ) -> httpx.Response:
+        """
+        Updates the labels of a flow run.
+        """
+
+        response = await self._client.patch(
+            f"/flow_runs/{flow_run_id}/labels", json=labels
+        )
+        response.raise_for_status()
+        return response
```

Comment:
Let's not return the response from here since it looks like it isn't needed.
```suggestion
    async def update_flow_run_labels(
        self, flow_run_id: UUID, labels: KeyValueLabelsField
    ) -> None:
        """
        Updates the labels of a flow run.
        """

        response = await self._client.patch(
            f"/flow_runs/{flow_run_id}/labels", json=labels
        )
        response.raise_for_status()
```